Geometry/HGCalCommonData/interface/HGCalWaferMask.h

```diff
@@ -0,0 +1,32 @@
+#ifndef HGCalCommonData_HGCalWaferMask_h
+#define HGCalCommonData_HGCalWaferMask_h
+
+/** \class HGCalWaferMask
+ *
+ * this class determines the masking of wafers to mimic partial wafers
+ *
+ *  $Date: 2019/01/15 00:06:50 $
+ * \author Sunanda Banerjee, Fermilab <sunanda.banerjee@cern.ch>
+ *
+ */
+
+#include <cmath>
+#include <vector>
+
+class HGCalWaferMask {
+public:
+  HGCalWaferMask() = default;
+
+  static bool maskCell(int u, int v, int N, int ncor, int fcor, int corners);
+  static bool goodCell(int u, int v, int N, int type, int rotn);
+  static std::pair<int, int> getTypeMode(const double& xpos,
+                                         const double& ypos,
+                                         const double& r1,
+                                         const double& R1,
+                                         const double& rin,
+                                         const double& rout,
+                                         const int& NW,
```

Comment:
Some of these parameter names should be changed. One-character names should only be used for loop indices. Names should start with lowercase. And please, please, do not use two names (r1 and R1) that only differ in capitalization.
Naming rules link: https://github.com/cms-sw/cms-sw.github.io/blob/master/cms_coding_rules.md#2----naming-rules-1

Reply:
Taken care of this

Reply:
"NW" is still not a recommended variable name (should be lowercase)